internal/templates/02-import.yaml.tmpl

```diff
@@ -8,6 +8,7 @@ commands:
 - command: sleep 10
 - command: ${KUBECTL} scale deployment crossplane -n ${CROSSPLANE_NAMESPACE} --replicas=1
 - script: ${KUBECTL} -n ${CROSSPLANE_NAMESPACE} get deploy --no-headers -o custom-columns=":metadata.name" | grep "provider-" | xargs ${KUBECTL} -n ${CROSSPLANE_NAMESPACE} scale deploy --replicas=1
+- script: ${KUBECTL} -n ${CROSSPLANE_NAMESPACE} get deploy --no-headers -o custom-columns=":metadata.name" | grep "provider-" | xargs ${KUBECTL} -n ${CROSSPLANE_NAMESPACE} wait deploy --for=condition=available --timeout=60s
```

Comment:
Our local tests were fine but we observed a race condition in https://github.com/crossplane-contrib/provider-upjet-azuread/pull/123. Hopefully, this synchronization statement will resolve it.

Reply:
Looks like we will need to update the test manifest also...